storage-transfer/system-test/transfer.test.js

```diff
@@ -16,49 +16,46 @@
 /* eslint no-empty: 0 */
 'use strict';
 
-const {Storage} = require(`@google-cloud/storage`);
+const {Storage} = require('@google-cloud/storage');
 const storage = new Storage();
-const test = require(`ava`);
-const tools = require(`@google-cloud/nodejs-repo-tools`);
-const uuid = require(`uuid`);
+const assert = require('assert');
+const tools = require('@google-cloud/nodejs-repo-tools');
+const uuid = require('uuid');
 
-const program = require(`../transfer`);
+const program = require('../transfer');
 
 const firstBucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
 const secondBucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
 
 let jobName;
-const date = `2222/08/11`;
-const time = `15:30`;
-const description = `this is a test`;
-const status = `DISABLED`;
+const date = '2222/08/11';
+const time = '15:30';
+const description = 'this is a test';
+const status = 'DISABLED';
 
-test.before(tools.checkCredentials);
-test.before(async () => {
+before(async () => {
+  tools.checkCredentials();
```

Comment:
We have just dropped that line from all the samples, and the console stubbing.  It's fine. Should we drop it here too?